src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -0,0 +1,47 @@
+package com.example.bookshop.controller;
+
+import com.example.bookshop.dto.BookDto;
+import com.example.bookshop.dto.CreateBookDto;
+import com.example.bookshop.service.BookService;
+import java.util.List;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/books")
```

Comment:
extract /api to common project prefix 
`server.servlet.context-path` property can do this